powerhome/playbook
# [PLAY-382] -Filter Doc Popover Close On Click

#### Screens
![Screen Shot 2022-10-13 at 10 42 11 AM](https://user-images.githubusercontent.com/73710701/195628285-1b185b39-1a17-4379-8ff6-e6a73352f7bd.png)


#### Breaking Changes
No, only doc change to close all doc examples when Apply button is clicked.

#### Runway Ticket URL

[RUNWAY STORY](https://nitro.powerhrg.com/runway/backlog_items/PLAY-382)

#### How to test this

review in milano env, make sure all doc examples popover close on click of Apply button in popover

#### Checklist:
